ganga_ai/config.py

```diff
@@ -1,6 +1,29 @@
+import configparser
+import os
+
 class Config:
+    _config_file_path = ".gangaairc"
+    _default_system_prompt = "Your task is to help users write and diagonize the python code they write at a Ipython terminal. Use markdown to format the responses. If the user is interactintg with you the message starts with %%assisst else the user is just running commands."
```

Comment:
interactintg -> interacting

Reply:
Will correct the typo. Thx for pointing it out.